server/app/src/main/java/com/ams/app/repository/AbsentdataRepository.java

```diff
@@ -0,0 +1,28 @@
+package com.ams.app.repository;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+import com.ams.app.model.Absentdata;
+import com.ams.app.model.Student;
+
+@Repository
+@Transactional
+public interface AbsentdataRepository extends JpaRepository<Absentdata, Integer>{
+    Absentdata findById(int Id);
+    
+    @Query(value = "SELECT * FROM Absentdata pd WHERE pd.aid = :aid AND pd.sid = :sid", nativeQuery = true)
+    Absentdata findByAidAndSid(@Param(value = "aid") int aid, @Param(value = "sid") int sid);
```

Comment:
We don't need query for this. This can be achieved by just 
`findByAidAndSid(int aid, int sid)`